sc-platform-api/src/documentation/article/create.ts

```diff
@@ -0,0 +1,23 @@
+import { ApiDocumentationParams } from '../decorators';
+import { DefaultUnauthorizedResponse } from '../default/unauthorized.response';
+
+export const Create: ApiDocumentationParams = {
+  isAuth: true,
+  ok: true,
+  unauthorized: DefaultUnauthorizedResponse,
```

Comment:
You can simply set this property to `true` as this response is set as default in `ApiEndpoint` decorator logic
```suggestion
  unauthorized: true,
```